### DISCUSSION TITLE
sentencepiece installation error

### DISCUSSION THREAD

**Discussion by taaha** [ParisNeo/Gpt4All-webui#18] [Q&A]
Really excited about this GUI but getting the following error on linux when I run install.sh
![image](https://user-images.githubusercontent.com/5251294/230713698-af88fd33-6ac8-4a9f-b8af-ae979ea0242d.png)
Any ideas how could this be solved. Thanks

**Comment by ParisNeo** [comment]
You are not the first to have had this issue. It seams that there is a problem with sentencepiece installation for ubuntu.
NOTE: the official repo has been moved to nomic-ai's github. So please go there to get the latest version of the software:
https://github.com/nomic-ai/gpt4all-ui

**Reply by Adrianogb** [reply]
Windows 10 Pro, 8GB ram, PowerShell, Python 3.10.9
Upgrading schema to version 3... 
ZZZZZZZZzzzzzzz
